assets/prototype/application/services/apollo/mutations/useEntityMutation.js

```diff
@@ -62,9 +69,9 @@ const useEntityMutation = (type, id = '') => {
 		 */
 		return (proxy, result) => {
 			// e.g. "createDatetime", "updateTicket"
-			const mutationName = `${mutationType.toLowerCase()}${type}`;
+			const mutationName = `${mutationType.toLowerCase()}Espresso${type}`;
 			// Example result: { data: { deletePrice: { price : {...} } } }
-			const path = ['data', mutationName, type.toLowerCase()];
+			const path = ['data', mutationName, `espresso${type}`];
```

Comment:
should we do something here to ensure incoming entity types are Capitalized?

Reply:
yes, an additional guard would be nice

Reply:
I agree